Update documentation with correction

vaerge opened the pull request:
the documentation for handling access through service principal had a small error

azurecla commented:
Hi __@jboeshart__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@jboeshart : Thanks for your contribution to the Azure documentation! The author, @MikeRayMSFT, has been notified to review your proposed change.

cjgronlund commented:
@MikeRayMSFT @JennieHubbard 
Please let me know what I should do with this PR. Thanks!

MikeRayMSFT commented:
@cjgronlund @JennieHubbard 
I'm looking over the changes now. Thanks for raising it to my attention.

MikeRayMSFT commented:
@jboeshart, thank you very much for submitting this change. Your submission was spot on. I have been working on an update for the document in a different pull request so I manually integrated your change. Thanks again for helping us make Azure docs better.

MikeRayMSFT commented:
@cjgronlund,

Can we please close this pull request? I already had a PR open for other changes in this same space. I have incorporated the changes proposed by @jboeshart and signed off on the other request in the private repo.

JennieHubbard commented:
@jboeshart per @MikeRayMSFT request please close this PR

jboeshart commented:
Closing since @MikeRayMSFT already included these changes in his PR. Thanks all!

Commit:
Update documentation with correction

the documentation for handling access through service principal had a small error
